src/models/Schema.ts

```diff
@@ -51,74 +51,80 @@ export class Schema extends CommonSchema<Schema | boolean>{
      * @param object to transform
      * @returns CommonModel instance of the object
      */
-    static toSchema(object: Object) : Schema | boolean{
-        if(typeof object === "boolean") return object;
-        let schema = new Schema();
-        schema = Object.assign(schema, object);
-        schema = CommonSchema.transformSchema(schema, Schema.toSchema) as Schema;
+    static toSchema(object: Object): Schema | boolean {
+        const seenSchemas: Map<any, Schema> = new Map();
+        const callback = (object: Object): Schema | boolean => {
```

Comment:
Also here we can remove callback and add `seenSchemas` as second argument of `toSchema` function

Reply:
Good catch!